sdc/utilities/prange_utils.py

```diff
@@ -54,36 +54,19 @@ def get_pool_size_impl():
     return get_pool_size_impl
 
 
+@sdc_register_jitable
 def get_chunks(size, pool_size=0):
     if pool_size == 0:
```

Comment:
Suggestion - remove this check and always pass `pool_size` explicitly.

Pure functions are good

Reply:
Done.

Reply:
So now we should explicitly call `get_pool_size()` and pass it to `get_chunks()`. Am I correct?

Reply:
Yes

Reply:
Seems it's not so convenient.

Reply:
I can introduce function like `parallel_chunks(N)` which will do call `get_chunks(N, get_pool_size())`. @Hardcode84, @densmirn ok?

Reply:
I like the idea.

Reply:
Done